src/main/java/com/bookclub/service/BookServiceManager.java

```diff
@@ -0,0 +1,45 @@
+package com.bookclub.service;
+
+import com.bookclub.dao.BookDAO;
+import com.bookclub.model.Book;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class BookServiceManager {
+
+    private BookDAO bookDAO;
```

Comment:
```suggestion
    private IBookAO bookDAO;
```
And change the variable name to be more generic. We want to be able to test against this with a MOA.